tests/functional/test_data_classes.py

```diff
@@ -544,9 +544,19 @@ def test_dynamo_attribute_value_null_value():
     attribute_value = AttributeValue(example_attribute_value)
 
     assert attribute_value.get_type == AttributeValueType.Null
+    assert attribute_value.null_value is None
     assert attribute_value.null_value == attribute_value.get_value
 
 
+def test_dynamo_attribute_value_null_value_invalid():
+    example_attribute_value = {"NULL": False}
```

Comment:
shouldn't this be `True`?

https://github.com/boto/boto3/blob/7a10e21c452e0cb0a6e3d245f7389f1500d2abcf/tests/unit/dynamodb/test_types.py#L69

Reply:
@michaelbrewer - We are deserializing,  `{"NULL": False}` should never happen (so should we raise a `ValueError` or just ignore that?)

Reply:
@heitorlessa - i can remove that test, i just wanted it to be clear that we don't valid 

```
    @property
    def null_value(self) -> None:
        """An attribute of type Null.

        Example:
            >>> {"NULL": True}
        """
        return None
```

Reply:
which is the same as:

```
    def _deserialize_null(self, value):
        return None
```